save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -105,14 +141,30 @@ internal class OrganizationController(
      */
     @PostMapping("/{organizationName}/update")
     @Suppress("UnsafeCallOnNullableType")
-    fun updateOrganization(@RequestBody organization: Organization, authentication: Authentication): Mono<StringResponse> = Mono
-        .just(organization)
-        .filter { organizationPermissionEvaluator.hasGlobalRoleOrOrganizationRole(authentication, it.name, Role.OWNER) }
-        .map {
-            organizationService.updateOrganization(organization)
+    fun updateOrganization(
+        @RequestBody organization: Organization,
+        authentication: Authentication,
+    ): Mono<StringResponse> = Mono.just(
+        organization.name
```

Comment:
yeah, @Cheshiriks is right - this is very strange - you do the following:
1) Get organization object
2) Get name from it 
3) Find this organization again by name 
4) Save Object 
5) get a name from an object 
6) Find object again 
7) Save results

Reply:
Fixed this. Hope everything else is alright